jbpm-installer/src/test/java/org/jbpm/persistence/scripts/UpgradeScriptsTest.java

```diff
@@ -20,23 +20,40 @@
 import java.io.IOException;
 import java.sql.SQLException;
 import java.text.ParseException;
-
-import org.jbpm.persistence.scripts.util.TestsUtil;
 import org.junit.Assert;
 import org.junit.Test;
 import org.kie.api.runtime.process.ProcessInstance;
 import org.kie.internal.runtime.StatefulKnowledgeSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.jbpm.persistence.scripts.TestPersistenceContext.createAndInitContext;
 
 /**
  * Contains tests that test database upgrade scripts.
  */
-public class UpgradeScriptsTest {
+public class UpgradeScriptsTest extends ScriptsBase{
```

Comment:
I don't want to be nitpicking here, though I just noticed a missing space near the left brace. BTW we have preconfigured files with our KIE Java Conventions [here](https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration) in case you want IDE to do it automatically.

Reply:
no problem! it's better to fix it before merging, done